R/RunAnalyses.R

```diff
@@ -825,19 +848,24 @@ addPsToStudyPopForSubset <- function(subset, outputFolder) {
   ps <- readRDS(file.path(outputFolder, subset$sharedPsFile[1]))
 
   addToStudyPop <- function(i) {
-    refRow <- subset[i, ]
+    refRow <- subset[i,]
     studyPop <- readRDS(file.path(outputFolder, refRow$studyPopFile))
     studyPop <- addPsToStudyPopulation(studyPop, ps)
-    saveRDS(studyPop, file.path(outputFolder, refRow$psFile))
+    tryCatch({
```

Comment:
Is the error message thrown by saveRDS not informative enough? If so, maybe we should have a generic wrapper for saveRDS in CohortMethod that we always use?